app.rb

```diff
@@ -22,15 +22,18 @@ class MakersBnB < Sinatra::Base
 
   get '/spaces' do
     @user = User.find(session[:user_id])
-    @spaces = Space.all
+    if @user 
+    @spaces = Space.spaces_listed(session[:user_id])
+    end
```

Comment:
Current approach is that spaces is not visible if not logged in. That's ok. Justr confirming that this is the approach.